src/filters/simple-filter-directive.js

```diff
@@ -183,13 +183,23 @@ angular.module('patternfly.filters').directive('pfSimpleFilter', function () {
         return foundFilter !== undefined;
       };
 
+      $scope.enforceSingleSelect = function (filter) {
+        if (filter.type === 'select') {
+          _.remove($scope.config.appliedFilters, {title: filter.title, type: filter.type});
+        }
```

Comment:
I don't think we need type here. Any filters with that title should have the same type, otherwise the same title (field) would show up in the list of filters more than once.